pypowsybl/network/impl/network.py

```diff
@@ -252,30 +253,58 @@ def write_network_area_diagram_svg(self, svg_file: PathOrStr, voltage_level_ids:
                                        low_nominal_voltage_bound: float = -1,
                                        edge_name_displayed: bool = False) -> None:
         """
+        .. deprecated:: 1.1.0
+        Use :class:`write_network_area_diagram_svg` with  `NadParameters` instead.
```

Comment:
It seems to me that it  should be `Use :class:  write_network_area_diagram ...` instead (without the svg)